src/main/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseController.java

```diff
@@ -263,4 +267,25 @@ public ResponseEntity<CaseDetails> updateResponseAsViewed(
         CaseDetails finalCaseDetails = applicationService.updateTribunalResponseAsViewed(authorization, request);
         return ok(finalCaseDetails);
     }
+
+    /**
+     * Submits a stored Claimant Application.
+     *
+     * @param authorization jwt of the user
+     * @param request       the request object which contains the claimant application passed from sya-frontend
+     * @return the new updated case wrapped in a {@link CaseDetails}
+     */
+    @PutMapping("/submit-stored-claimant-application")
```

Comment:
Unfortunately this may be the straw that broke the camel's back. This really should be in a new controller as this class has gotten too large. It will mean you need to change the frontend call too to point at a different endpoint